styles/general.css

```diff
@@ -0,0 +1,87 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500&display=swap');
+
+*::before,
+*::after {
+  margin: 0;
+}
+
+body {
+  background-color:  #00040F;
+  font-family: 'Poppins', sans-serif;
+  color: white;
+  margin: 0;
+
+  padding: 32px 24px;
+}
```

Comment:
- [ ] For the background color please make sure that it matches the original design 

- Your design 👇 

![image](https://user-images.githubusercontent.com/75126481/219377909-ebe6e894-ea47-48a6-a228-19a19059dfb8.png)

- Origina design 👇 

![image](https://user-images.githubusercontent.com/75126481/219378099-b12fb80d-4e20-4b79-ba3b-034a88fb9bd8.png)